packages/cli/src/codegen/schema.ts

```diff
@@ -295,19 +299,55 @@ Suggestion: add an '!' to the member type of the List, change from '[${baseType}
     return gqlType.getIn(['name', 'value']);
   }
 
-  _typeFromGraphQl(gqlType: immutable.Map<any, any>, nullable = true): any {
+  _typeFromGraphQl({
+    gqlType,
+    nullable = true,
+    entityDef,
+    fieldDef,
+  }: {
+    gqlType: immutable.Map<any, any>;
+    nullable?: boolean;
+    entityDef: any;
+    fieldDef: immutable.Map<any, any>;
```

Comment:
these are used so we can hint users what to change to make migration simpler : )

Reply:
I guess we can go one extra step and do `filename:location` this way user can click into error, kinda like how we ESlint or GraphQL codegen would do but that would require more refactor to pass the context of file around 🥲